.github/workflows/bench.yml

```diff
@@ -316,27 +316,24 @@ jobs:
                   transmogrify "${server_cmd[$server]}" "$cc" "$pacing" "${neqo_flags[$client]}"
                   FILENAME="$client-$server$EXT"
                   SERVER_CMD="$CMD"
-                  # shellcheck disable=SC2086
-                  nice -n -20 setarch --addr-no-randomize cset proc --set=cpu2 --exec ${CMD/ / -- } &
-                  transmogrify "${client_cmd[$client]}" "$cc" "$pacing" "${neqo_flags[$server]}"
-                  # shellcheck disable=SC2086
+                  # pkill only matches on the first 15 characters of the command?!
+                  SERVER_TAG="$(basename "$(echo "${server_cmd[$server]}" | cut -f1 -d' ')" | cut -c1-15)"
```

Comment:
You could always use `$&` instead.  It's possible that the long command line is causing this to hit some limit.

Reply:
Oh, that doesn't work.  The pkill invocation is not made by the shell, but by hyperfine itself.  For that to work, you'd need a wrapper that writes the PID to a file.